crates/consensus-logic/src/csm/ctl.rs

```diff
@@ -5,7 +5,7 @@ use strata_state::sync_event::SyncEvent;
 use tokio::sync::{mpsc, oneshot};
 use tracing::*;
 
-use crate::message::CsmMessage;
+use super::message::CsmMessage;
```

Comment:
This change is not needed

Reply:
It should be, I moved the message module too, right?